spec/tests/angularComponent/ngcOmniboxControllerSpec.js

```diff
@@ -8,6 +8,7 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
 
     omniboxController = new NgcOmniboxController();
     omniboxController._suggestionElements = [fakeEl, fakeEl, fakeEl, fakeEl];
+    omniboxController.isSelectable = () => true;
```

Comment:
Needing to do this makes me nervous. If app-maker does **not** provide anything to `is-selectable`, then by default everything should be selectable, right?